### ISSUE TITLE
cannot reproduce core

### ISSUE THREAD

**Issue by dgcovell** [Tony607/Keras_Deep_Clustering#13]
Trying to implement Keras_Deep_Clustering in python. Results indicate ~10% accuracy versus expected accuracy of ~52%.
Not sure whether I am using proper metrics. I have installed metrics.py (as in blog) and used sklearn.metrics.accuracy_score but
am not getting above 10%.
Also tried in jupyter norebook w/o success.

Any advice would be appreciated.

Thanks
